recipe/meta.yaml

```
@@ -30,19 +31,20 @@ build:
 
 requirements:
   host:
-    - python
     - pip
-    - setuptools
+    - python
+    - setuptools >=62
```

setuptools must be >=70.1 to remove wheel
```suggestion
    - wheel
    - setuptools >=62
```